ftplugin/vimwiki/taskwiki.vim

```diff
@@ -103,6 +110,8 @@ if !exists('g:taskwiki_suppress_mappings')
                         let maplocalleader = '\t'
                 endif
         endif
+        " Override vimwiki's <C-Space>
+        nnoremap <silent><buffer> <C-Space> :TaskWikiDone<CR>
```

Comment:
Why this second mapping? Isn't the one on line 100 enough?

Reply:
That's a good question... I didn't understand very well this part at first but now re-reading it I think you are right. I just removed these lines and launched a new build[1]. The tests I added pass so probably I messed that up, thanks for pointing out!
Updating PR!

[1]: https://travis-ci.org/github/aaronfc/taskwiki/builds/699457085